payments/views.py

```diff
@@ -53,24 +53,21 @@ def dispatch(self, *args, **kwargs):
         return super(CSRFExemptMixin, self).dispatch(*args, **kwargs)
 
 
-class UserIdentityVerifiedMixin(UserPassesTestMixin):
-    login_url = reverse_lazy('identity')
+class UserHasAcceptedTermsMixin(UserPassesTestMixin):
+    login_url = reverse_lazy('terms')
     redirect_field_name = None
 
     def test_func(self):
-        user_account = self.request.user.account()
-        if not user_account:
-            self.login_url = reverse_lazy('bank')
-            return False
-        return user_account.identity_verified()
+        return self.request.user.accepted_terms()
 
 
-class UserHasAcceptedTermsMixin(UserPassesTestMixin):
-    login_url = reverse_lazy('terms')
+class UserIdentityVerifiedMixin(UserHasAcceptedTermsMixin):
+    login_url = reverse_lazy('identity')
     redirect_field_name = None
 
     def test_func(self):
```

Comment:
Super does not work, in fact trying to stack mixins doesn't work.  By definition each mixins are setting the login_url property.  I decide to write one mixin that test both conditions.